gulpfile.babel.js

```diff
@@ -57,29 +57,37 @@ gulp.task('serve', () => {
       webpackHotMiddelware(compiler)
     ]
   });
-});
+}
 
 /**
- * Alias for webpack task
+ * Uploads the built widgets to a EVRYTHNG File entity.
  */
-gulp.task('build', gulp.series('webpack'));
+function publish() {
+  return gulp.series(compile, deployWidgets)();
```

Comment:
don't use gulp series/parallel methods in tasks definitions. use only in task registration below (that's where you orchestrate the task flow using the various subtasks):

```javascript
gulp.task('publish', gulp.series(build, publish));
```